### DISCUSSION TITLE
systemctl failed start Writefreely Instance (permission denied)

### DISCUSSION THREAD

**Discussion by DocWho22** [writefreely/writefreely#552] [Questions / Support] [answered]
### Describe the bug
As I ran `systemctl status writefreely`, I got this:
```bash
[root@instance-20220429-1753 writefreely]# systemctl status writefreely
● writefreely.service - WriteFreely Instance
   Loaded: loaded (/etc/systemd/system/writefreely.service; enabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Thu 2022-05-05 08:16:52 CST; 3s ago
  Process: 59535 ExecStart=/var/www/changedtomydomain/writefreely/writefreely (code=exited, status=203/EXEC)
 Main PID: 59535 (code=exited, status=203/EXEC)

May 05 08:16:51 instance-20220429-1753 systemd[1]: writefreely.service: Main process exited, code=exited, status=203/EXEC
May 05 08:16:51 instance-20220429-1753 systemd[1]: writefreely.service: Failed with result 'exit-code'.
May 05 08:16:52 instance-20220429-1753 systemd[1]: writefreely.service: Service RestartSec=100ms expired, scheduling restart.
May 05 08:16:52 instance-20220429-1753 systemd[1]: writefreely.service: Scheduled restart job, restart counter is at 5.
May 05 08:16:52 instance-20220429-1753 systemd[1]: Stopped WriteFreely Instance.
May 05 08:16:52 instance-20220429-1753 systemd[1]: writefreely.service: Start request repeated too quickly.
May 05 08:16:52 instance-20220429-1753 systemd[1]: writefreely.service: Failed with result 'exit-code'.
May 05 08:16:52 instance-20220429-1753 systemd[1]: Failed to start WriteFreely Instance.
```
But when I ran `/var/www/changedtomydomain/writefreely/writefreely`, the website **worked**!

SO in the end I used `tmux` to run writefreely.

### Steps to reproduce (if necessary)
My service file:
```conf
[root@instance-20220429-1753 writefreely]# cat /etc/systemd/system/writefreely.service
[Unit]
Description=WriteFreely Instance
After=syslog.target network.target mysql.service

[Service]
Type=simple
StandardOutput=syslog
StandardError=syslog
WorkingDirectory=/var/www/changedtomydomain/writefreely
ExecStart=/var/www/changedtomydomain/writefreely/writefreely
Restart=always

[Install]
WantedBy=multi-user.target
```
I also tried `chmod -x /var/www/changedtomydomain/writefreely/writefreely`, but it didn't make any difference.

### Expected behavior
What should've happened?

### Application configuration
 - **Single mode or Multi-user mode?**  Multi-user
 - **Database?** mysql
 - **Open registration?** yes
 - **Federation enabled?** yes

**Version or last commit**:
latest version [v0.13.1](https://github.com/writefreely/writefreely/releases/tag/v0.13.1)

### My System
`Oracle RHEL 8 aarch64`

**Comment by thebaer** [comment, marked as answer]
Can you share the application logs from `journalctl`? That will probably indicate what's going wrong when you try to start it up.

**Reply by DocWho22** [reply]
Thanks for reply. Very sorry that I stopped writefreely because I found it did not fit me. So I could not give the logs from `journalctl`.

I guess it may because SELINUX. I run `systemctl` with `root`. But I created a user `writefreely` and download writefreely with it. So it has this problem.

Thanks for your reply! Have a nice day and stay healthy.

Best regards.
